shared/domains/chelonia/chelonia-utils.js

```diff
@@ -0,0 +1,13 @@
+import sbp from '@sbp/sbp'
+
+export default (sbp('sbp/selectors/register', {
+  'chelonia/kv/queuedSet': ({ contractID, key, data, onconflict, encryptionKeyName = 'cek', signingKeyName = 'csk' }) => {
+    return sbp('chelonia/queueInvocation', contractID, () => {
+      return sbp('chelonia/kv/set', contractID, key, data, {
+        encryptionKeyId: sbp('chelonia/contract/currentKeyIdByName', contractID, encryptionKeyName),
+        signingKeyId: sbp('chelonia/contract/currentKeyIdByName', contractID, signingKeyName),
+        onconflict
+      })
+    })
+  }
```

Comment:
Request:

- could be this placed next to the the existing selector `'chelonia/kv/set'` so that it's easier to see?
- could you please add a comment above it explaining why usually calling this version is preferred and what this queueing accomplishes?

Reply:
> could be this placed next to the the existing selector 'chelonia/kv/set' so that it's easier to see?

I think `chelonia.js` is already cluttered as it is. The reason for putting this in a different file is also that IMHO this belongs in a possibly optional library as it makes some assumptions about which keys are defined (i.e., the csk and the cek) which isn't done in Chelonia currently.

When encryptedAction is moved into Chelonia, it would also be placed in this new file, because it also is a 'suggested' way of doing a common action, but not something that applies to all situations.

> Could you please add a comment above it explaining why usually calling this version is preferred

Sure, although calling this version isn't 'preferred', just less much verbose in a common situation. I can explain that.

Reply:
Sure, if you want to leave it in that file, then please add a comment above `'chelonia/kv/set'` to tell people to look at `'chelonia/kv/queuedSet'` which is preferable (and where to find that).